### DISCUSSION TITLE
i just want a self contained program that can merge to font file

### DISCUSSION THREAD

**Discussion by ghost** [starship/starship#1123] [General]
https://github.com/starship/starship/issues/1120

**Comment by ghost** [comment]
better if written in rust

**Comment by ghost** [comment]
i want to merge 'source code pro' with 'seti'

**Comment by johnletey** [comment]
@mobop I'm not entirely quite sure if there is a way to do this.

However, as I mentioned before, there is a NerdFont patch for the Source Code Pro font: https://github.com/ryanoasis/nerd-fonts/tree/master/patched-fonts/SourceCodePro
